cogs/tags/tags.py

```diff
@@ -385,7 +407,7 @@ async def dump(self, ctx: Context):
 
     @tag.command(name="add", aliases=["create"])
     @commands.guild_only()
-    @role_or_mod_or_permissions(isAllowedRole(role), manage_messages=True)
+    @roles_or_mod_or_permissions(roles=allowed_roles, manage_messages=True)
```

Comment:
Here and elsewhere, match kwarg arg in decorator.
```suggestion
    @roles_or_mod_or_permissions(allowed_roles=allowed_roles, manage_messages=True)
```